src/main/java/org/sonatype/repository/conan/internal/proxy/matcher/StandardMatcher.groovy

```diff
@@ -0,0 +1,124 @@
+package org.sonatype.repository.conan.internal.proxy.matcher
+
+import org.sonatype.nexus.repository.view.Route.Builder
+import org.sonatype.nexus.repository.view.matchers.ActionMatcher
+import org.sonatype.nexus.repository.view.matchers.token.TokenMatcher
+import org.sonatype.repository.conan.internal.AssetKind
+
+import static org.sonatype.nexus.repository.http.HttpMethods.GET
+import static org.sonatype.nexus.repository.http.HttpMethods.HEAD
+import static org.sonatype.nexus.repository.view.matchers.logic.LogicMatchers.and
+import static org.sonatype.nexus.repository.view.matchers.logic.LogicMatchers.or
+import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.GROUP
+import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.PROJECT
+import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.STATE
+import static org.sonatype.repository.conan.internal.metadata.ConanMetadata.VERSION
+
+/**
+ * For {@link AssetKind#DOWNLOAD_URL} the server sends files in Group/Project/Version order
+ * for all other types it is in Version/Group/Project order
+ * @since 0.0.2
+ */
+class StandardMatcher
+    extends ConanMatcher
+{
+  public static final String NAME = "standard";
+
+  /**
+   * Matches on urls ending with download_urls
+   * @return
```

Comment:
We have a `@return` Javadoc tag that has no information.

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/15/commits/fa050ac837279430117d8f757735f1d66da2672e